src/main/java/net/dv8tion/jda/api/entities/ChannelField.java

```diff
@@ -21,35 +21,175 @@
+     *
+     * @see IPositionableChannel#getPosition()
+     * //todo-v5 if the calculation of IPositionableChannel#getPosition() changes, this may need modification too.
+     */
     POSITION("position", null), //Discord doesn't track Channel position changes in AuditLog.
 
+
+    //Text Specific
+
+    /**
+     * The topic of the channel.
+     *
+     * Limited to {@link BaseGuildMessageChannel Base Guild Channels} (and implementations).
+     *
+     * @see BaseGuildMessageChannel#getTopic()
+     */
     TOPIC("topic", AuditLogKey.CHANNEL_TOPIC),
+
+    /**
+     * The NSFW state of the channel.
+     *
+     * Limited to {@link BaseGuildMessageChannel Base Guild Channels} (and implementations).
+     *
+     * @see BaseGuildMessageChannel#isNSFW()
+     */
     NSFW("nsfw", AuditLogKey.CHANNEL_NSFW),
+
+    /**
+     * The state of slow mode in the channel.  This defines the minimum time between message sends.
+     *
+     * Limited to {@link TextChannel Text Channels}.
+     *
+     * @see TextChannel#getSlowmode()
+     */
     SLOWMODE("slowmode", AuditLogKey.CHANNEL_SLOWMODE),
 
-    PARENT("parent", AuditLogKey.CHANNEL_PARENT),
 
+    //Voice Specific
+
+    /**
+     * The bitrate (in bits per second) of the audio in this channel.
+     *
+     * For standard channels this is between 8000 and 96000.
+     *
+     * VIP servers extend this limit to 128000.
+     *
+     * Limited to {@link AudioChannel Audio Channels}.
+     *
+     * @see AudioChannel#getBitrate()
+     */
     BITRATE("bitrate", AuditLogKey.CHANNEL_BITRATE),
+
+    /**
+     * The region of the channel.
+     *
+     * Limited to {@link AudioChannel Audio Channels}.
+     *
+     * @see AudioChannel#getRegion()
+     * @see net.dv8tion.jda.api.Region
+     */
     REGION("region", null), //TODO-v5: JDA needs to add support for channel-specific audit log tracking
+
+    /**
+     * The maximum user count of this channel.
+     *
+     * Limited to {@link VoiceChannel Voice Channels}.
+     *
+     * @see VoiceChannel#getUserLimit()
+     */
     USER_LIMIT("userlimit", AuditLogKey.CHANNEL_USER_LIMIT),
 
+
     //Thread Specific
+
+    /**
+     * The auto archive duration of this channel.
+     *
+     * If the thread is inactive for this long, it becomes auto-archived.
+     *
+     * Limited to {@link ThreadChannel Thread Channels}.
+     *
+     * @see ThreadChannel#getAutoArchiveDuration()
+     */
     AUTO_ARCHIVE_DURATION("autoArchiveDuration", AuditLogKey.THREAD_AUTO_ARCHIVE_DURATION),
+
+    /**
+     * The archive state of this channel.
+     *
+     * If the channel is archived, this is true.
+     *
+     * Limited to {@link ThreadChannel Thread Channels}.
+     *
+     * @see ThreadChannel#isArchived()
+     */
     ARCHIVED("archived", AuditLogKey.THREAD_ARCHIVED),
+
+    /**
+     * The archive time of this channel.
+     *
+     * If the channel is archived, this timestamp will be when it was archived.
```

Comment:
This isn't actually true. This timestamp represents the last time that something about the archival info for this thread changed in any way. So, if the thread is archived, unarchived, or the AUTO_ARCHIVE_DURATION time is updated in some way.

Reply:
I've changed the description, just want to make sure the new one is correct